spring-boot-project/spring-boot-docs/src/docs/antora/modules/reference/pages/features/profiles.adoc

```diff
@@ -32,7 +32,7 @@ spring:
     default: "none"
 ----
 
-`spring.profiles.active` and `spring.profiles.default` can only be used in non-profile specific documents.
+`spring.profiles.active` and `spring.profiles.default` can only be used in non-profile-specific documents.
```

Comment:
this is a trivial point, so I don't have strong preferense or opinion. I will revert if I should